src/packages/console/src/pages/canvasPage.tsx

```diff
@@ -97,29 +66,28 @@ export const CanvasPage = (): React.ReactElement => {
   };
 
   return (
-    <div className={classes.root}>
+    <Stack direction={Direction.Vertical} isFullHeight={true}>
       <Helmet>
         <title>Canvas | Everypage Console</title>
       </Helmet>
-      <div className={classes.topBar}>
-        <Button
-          variant='outlined'
-          onClick={onStartOverClicked}
-        >
-          Start again
-        </Button>
-        <div className={classes.spacer} />
-        <div className={classes.topBarSignInBox}>
+      <Box variant='banner'>
+        <Stack direction={Direction.Horizontal} paddingHorizontal={PaddingSize.Wide} isFullWidth={true}>
```

Comment:
if you make the child (or content, im not sure) allignment center thaen the start again button will be a better size. Make the other alignment center so its in the middle